tests/unit_tests/stencil/core/test_functor_metafunctions.cpp

```diff
@@ -19,13 +19,13 @@ namespace gridtools {
                 using interval_t = interval<level<0, 1, 3>, level<2, -1, 3>>;
 
                 struct empty {};
-                static_assert(!is_valid_functor<empty, interval_t>::value, "");
+                //                static_assert(check_valid_apply_overloads<empty, interval_t>::value, "");
```

Comment:
```suggestion
```

Reply:
or reenable?

Reply:
The comment is not a typo here. Note that the traits is named `check_...` now but not `is_...`. This is because it compiles only if the condition is true. If the condition is false the inner `static_assert` is triggered.